.github/workflows/tests.yaml

```diff
@@ -69,15 +69,15 @@ jobs:
           echo '*** Updating brew'
           brew update
           echo '*** Installing LLVM deps'
-          brew install --force-bottle --only-dependencies llvm
+          brew install --force-bottle --only-dependencies llvm@15
           echo '*** Installing LLVM itself'
-          brew install --force-bottle --force --verbose llvm
-          echo '*** brew info llvm'
+          brew install --force-bottle --force --verbose llvm@15
```

Comment:
@jonmeow told me offline that `@version` is incompatible with `--force-bottle` -- homebrew only has a prebuilt package available for the current `stable` version -- and apparently you can't even name version `15` with `@15`, only with `@stable`. :(

Reply:
I might have been wrong on the @15 note, actually. But older versions typically aren't bottled by brew, so this would break immediately on future upgrades. I don't think we should specify a version to brew when it breaks on changes.

Reply:
Thanks for the explanation.

Reply:
@jonmeow Is this trick acceptable?

Reply:
> I might have been wrong on the @15 note, actually. But older versions typically aren't bottled by brew, so this would break immediately on future upgrades. I don't think we should specify a version to brew when it breaks on changes.

@jonmeow Is this trick acceptable?

Reply:
> I'm not clear what you're trying to do here. From my perspective, `brew --prefix llvm` is an API for getting the install path. With the new `rm` / `ln` change, why are you trying to delete it and replace it? What's the problem you're trying to solve?

Problem may occur when there are more than one brew LLVM. Some macOS image may provide a brew LLVM by default, which is not necessarily the newest version. Since we install the new version of LLVM on the top of it, this script links to the newest among all installed brew LLVMs.

Reply:
This is GitHub's image though, not an arbitrary system. I don't think that's going to happen here.

Reply:
I'll add, I also would expect there's a standard way to update brew symlinks if this were an issue. I view this a bit from a perspective of wanting to limit what kinds of potential problems we try to solve when we're working with a specific system (GitHub), but I'm particularly hesitant to modify brew files without using brew commands.

Reply:
> I'll add, I also would expect there's a standard way to update brew symlinks if this were an issue. I view this a bit from a perspective of wanting to limit what kinds of potential problems we try to solve when we're working with a specific system (GitHub), but I'm particularly hesitant to modify brew files without using brew commands.

I agree.

Reply:
@jonmeow I looked around and found that the [latest](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md) image looks like it comes with brew 14

Reply:
You may find my script is safe: if it was updated beforehand(if that is what you want to show?), then my script does no harm it would make zero effect.

Reply:
I in this context thought of the internal of Homebrew as some Ruby and a little of bit of script layer like this manner. Homebrew may not be over-complicated like we may think of some extra virtual filesystem(?) with their own linkage if you are in concern? And this solves the hard-coding issue you found problematic in:
> ... specify a version to brew when it breaks on changes.
Does Carbon prohibit a shell scripting?

Reply:
```
lrwxr-xr-x  1 jperkins  primarygroup  21 Sep 19 17:25 /Users/jperkins/.brew/opt/llvm -> ../Cellar/llvm/15.0.0
```

It's a symlink: that looks fine to me. What's the issue here?